korman/properties/modifiers/gui.py

```diff
@@ -330,3 +333,302 @@ def _create_moul_nodes(self, clickable_object, nodes, journalnode, age_name):
+                             items=[("LinkOut", "Standing", "The avatar steps up to the book and places their hand on the panel"),
+                                    ("FishBookLinkOut", "Kneeling", "The avatar kneels in front of the book and places their hand on the panel"),],
+                             default="LinkOut",
+                             options=set())
+
+    # Interactables
+    seek_point = PointerProperty(name="Seek Point",
+                                 description="The point the avatar will seek to before opening the Linking Book GUI",
+                                 type=bpy.types.Object,
+                                 poll=idprops.poll_empty_objects)
+    clickable_region = PointerProperty(name="Clickable Region",
+                                       description="The region in which the avatar must be standing before they can click on the Linking Book",
+                                       type=bpy.types.Object,
+                                       poll=idprops.poll_mesh_objects)
+    clickable = PointerProperty(name="Clickable",
+                                description="The object the avatar will click on to activate the Linking Book GUI",
+                                type=bpy.types.Object,
+                                poll=idprops.poll_mesh_objects)
+
+    # -- Path of the Shell options --
+    # Popup Appearance
+    book_cover_image = PointerProperty(name="Book Cover",
+                                       description="Image to use for the Linking Book's cover",
+                                       type=bpy.types.Image,
+                                       options=set())
+    book_scale_w = IntProperty(name="Book Width Scale",
+                               description="Width scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    book_scale_h = IntProperty(name="Book Height Scale",
+                               description="Height scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    link_panel_image = PointerProperty(name="Linking Panel",
+                                       description="Image to use for the Linking Panel",
+                                       type=bpy.types.Image,
+                                       options=set())
+
+    stamped_book = BoolProperty(name="Has Stamp",
+                                description="Enables a stamp on the page opposite the book's linking panel",
+                                default=False)
+    stamp_image = PointerProperty(name="Stamp Image",
+                                  description="Image to use for the stamp",
+                                  type=bpy.types.Image,
+                                  options=set())
+    stamp_x = IntProperty(name="Stamp Position X",
+                          description="Width scale",
```

Comment:
Scale?

Reply:
Whoops!  Copied and only partially edited!